EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -21,64 +24,72 @@
     EasyInventoryService easyInventoryService;
     //Endpoints for the categories
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
     //Retrieves all the categories from the database
-    @RequestMapping(value="/category/all", method = RequestMethod.GET)
+    @GetMapping(value="/category/all")
     public List<Category> getAllCategories(Model model){
         return easyInventoryService.getAllCategories();
     }
 
-    @RequestMapping(value="/category/{id}", method = RequestMethod.GET)
+    @GetMapping(value="/category/{id}")
     public Category getCategoryById(@PathVariable int id, Model model) throws Exception {
         Category category = easyInventoryService.getCategoryByCategoryId(id);
         model.addAttribute("singleCategory", category);
         return easyInventoryService.getCategoryByCategoryId(id);
     }
 
     //Adds a new value to the category table
-    @RequestMapping(value="/category/newcategory", method = RequestMethod.POST)
+    @PostMapping(value="/category/newcategory")
     public Category addCategory(@RequestBody Category category) throws Exception {
         return easyInventoryService.saveCategory(category);
     }
 
     //Deletes a category from the table
-    @RequestMapping(value="/category/deletecategory", method = RequestMethod.DELETE)
+    @DeleteMapping(value="/category/deletecategory")
```

Comment:
Changed @RequestMapping to @DeleteMapping to make the code more readable and clean